### ISSUE TITLE
Notifier not fetching all proposals due to RPC rate limits

### ISSUE THREAD

**Issue by johnrees** [blockworks-foundation/governance-ui#103]
The governance notifier is currently using the [public RPC endpoint](https://github.com/blockworks-foundation/governance-ui/blob/4df37f710ae3715648dc920f026efda961c95b5d/scripts/governance-notifier.ts#L23) and is being rate limited on the [getProgramAccounts](https://github.com/blockworks-foundation/governance-ui/blob/4df37f710ae3715648dc920f026efda961c95b5d/scripts/api.ts#L100) instruction.

There is no retry logic when this happens the data it uses is incomplete. I think it either needs to implement retry or throttling behaviour, or use a different endpoint.
 
## public rpc - api.mainnet-beta.solana.com
```
Unexpected response {"jsonrpc":"2.0","error":{"code":429,"message":"Too requests for a specific RPC call, contact your app developer or support@rpcpool.com."},"id":1}

-- countJustOpenedForVoting: 0, countVotingNotStartedYet: 19, countClosed: 43
```

## genesysgo rpc
`-- countJustOpenedForVoting: 0, countVotingNotStartedYet: 20, countClosed: 45`

## [theindex.io](https://theindex.io)
`-- countJustOpenedForVoting: 0, countVotingNotStartedYet: 20, countClosed: 45`

**Comment by johnrees** [comment]
NB: there is no noticeable speed difference between the requests made to theindex.io and other RPC services in this instance

**Comment by johnrees** [comment]
Ah, I just found something in the code that might fix it. BRB!

**Comment by mschneider** [comment]
yeah, we never bother in working around rate limits and always just set a custom RPC via env. feel free to add another default that works better.

**Comment by johnrees** [comment]
> yeah, we never bother in working around rate limits and always just set a custom RPC via env. feel free to add another default that works better.

before seeing this comment I posted PR that fixes rate limits here #104

or an faster alternative WIP PR if you're planning to use private RPC infra here #105
